health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfiguration.js

```
@@ -307,7 +307,13 @@ const FormulaConfiguration = ({ onSelect, category, customProps, formulas: initi
               <div>
                 <Card type="secondary">
                   <LabelFieldPair className="formula-label-field">
-                    <span style={{ marginRight: "0rem" }}>{`${t(formula.output)}`}
+                    <span>
+                      {`${t(formula.output)}`}
+                      {category === "CAMPAIGN_VEHICLES" || formula?.source === "CUSTOM" ? null : (
+                        <span className="icon-wrapper">
+                          <TooltipWrapper content={t(`FORMULA_MESSAGE_FOR_${formula.output}`)} children={<InfoBannerIcon fill={"#C84C0E"} />} />
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)